docs/how-to/uc-image-creation.rst

```diff
@@ -5,11 +5,141 @@ How to create a real-time Ubuntu Core image
 
     This guide assumes access to features available exclusively to `dedicated Snap Store`_ users.
 
-The :doc:`../how-to/uc-boot-parameters` describes the method for dynamically configuring the system for real-time processing.
-The configuration is an iterative process that is best done together with the expected workload. 
-Once satisfied with the configurations, it is time to prepare the operating system for going into production.
+Canonical builds and publishes Ubuntu Core images for a range of `supported platforms`_.
+However, these images include the generic Linux kernel. 
+In order to run Ubuntu Core with the real-time kernel, we need to build it ourselves. 
 
-This guide shows how to statically set the desired Kernel command-line options for the Ubuntu Core system.
+This guide shows how to build and Ubuntu Core image with the real-time kernel.
+We start by showing how to :ref:`build a vanilla image <ubuntu-core-image>` which is excellent for testing and tuning, and then illustrate how to :ref:`make a custom image <custom-ubuntu-core-image>` with production-ready kernel configurations.
+
+
+.. _ubuntu-core-image:
+
+Create an image with the real-time kernel
+-----------------------------------------
+
+To do this we need to describe the image content and then use a tool to build it.
+
+.. _model-assertion:
+
+Create the model assertion
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The model assertion is a digitally signed document that describes the content of the Ubuntu Core image.
+Read the `model assertion`_ documentation before continuing.
+
+Below is an example model assertion in YAML, describing a `core22` Ubuntu Core
+image:
+
+.. literalinclude :: uc-image-creation/model.json
+   :language: json
+
+Inside an empty directory, create a file named ``model.json`` with the above content.
+
+Change the following:
+
+- ``authority-id``, ``brand-id`` to your developer ID, since this is custom model. Use ``snapcraft whoami`` command to get your developer ID.
+- ``timestamp`` to a RFC3339 formatted string, within you signing key's validity. Use ``date -Iseconds --utc`` command to use the current time.
+- ``store`` to your dedicated Snap Store ID.
+- ``model`` to something representative of your model.
+
+The ``snaps`` array is a list of snaps that get included in the image.
+In that list, the ``realtime-kernel`` snap contains the realtime Linux kernel.
+Here you can add any other snaps, including for example your real-time applications.
+
+Next, we need to sign the model assertion.
+Refer to the guide on `signing model assertion`_ for details on how to sign the model assertion. 
+
+Here are the needed steps:
+
+1) Create and register a key
+
+
+.. code-block:: shell
+
+    snapcraft create-key realtime-ubuntu
+    snapcraft register-key realtime-ubuntu
+
```

Comment:
Here, you might want to add a brief explanation:

- Creating a key refers to generating a GPG key.
- Registering the key means uploading the public GPG key to the Snapcraft Store account.

Reply:
I'd leave this to the respective documentation mentioned right above:

> Refer to the guide on [signing model assertion](https://ubuntu.com/core/docs/sign-model-assertion) for details on how to sign the model assertion. 
> 
> Here are the needed steps: